utils/KoalaDBManager.py

```diff
@@ -129,13 +131,88 @@ def create_base_tables(self):
         welcome_message text
         );"""
 
+        sql_create_guild_setup_table = """
+        CREATE TABLE IF NOT EXISTS GuildSetupStatus(
+        guild_id integer NOT NULL PRIMARY KEY, 
+        accepted_setup BOOLEAN NOT NULL CHECK (accepted_setup IN (0, 1))
+        );
+        """
+
+        sql_create_guild_dm_email_list_status_table = """
+        CREATE TABLE IF NOT EXISTS GuildDMEmailListStatus(
+        guild_id integer NOT NULL PRIMARY KEY,
+        dm_email_list_status BOOLEAN NOT NULL CHECK (dm_email_list_status IN (0, 1))
+        );
+        """
+
         self.db_execute_commit(sql_create_guild_welcome_messages_table)
         self.db_execute_commit(sql_create_koala_extensions_table)
         self.db_execute_commit(sql_create_guild_extensions_table)
+        self.db_execute_commit(sql_create_guild_setup_table)
+        self.db_execute_commit(sql_create_guild_dm_email_list_status_table)
+
+    def insert_setup_status(self, guild_id):

```

Comment:
Please add docstrings, only focus on methods you created as I'll make an issue to make sure all classes have proper docstrings

Reply:
Done